lib/proxy.js

```diff
@@ -20,7 +20,8 @@ var httpProxy = require('http-proxy');
 var env = {
 	noProxy: process.env.NO_PROXY || process.env.no_proxy,
 	httpProxy: process.env.HTTP_PROXY || process.env.http_proxy,
-	httpsProxy: process.env.HTTPS_PROXY || process.env.https_proxy
+	httpsProxy: process.env.HTTPS_PROXY || process.env.https_proxy,
+	remote_location: process.env.REMOTE_LOCATION || process.env.remote_location
```

Comment:
Can you change this to `remoteLocation` to be consistent with the other variables?